src/components/Notes.js

```diff
@@ -0,0 +1,51 @@
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid'
+import Paper from '@mui/material/Paper'
+import EditIcon from '@mui/icons-material/Edit';
+import DeleteIcon from '@mui/icons-material/Delete';
+import Typography  from '@mui/material/Typography';
+import IconButton from '@mui/material/IconButton';
+import Stack from '@mui/material/Stack';
+import '../App.css';
+
+const Notes = ({note}) => {
```

Comment:
since this component works only with the single note, it is misleading to call it `Notes`, call it with something more appropriate